locales/en/common.json

```diff
@@ -1,36 +1,100 @@
 {
   "404_page_does_not_exist": "404: That page doesn’t exist",
+  "accepted": "Creation pending",
+  "adjust_your_filters_and_try_again": "Adjust your filters and try again.",
+  "ams_service_down_message": "We are unable to create a new Service Registry instance at this time. Try again later.",
```

Comment:
Should this include the word 'Please' as well? There was similar discussion for kafka instance creation as well.